src/components/header/__test__/Header.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { describe, expect, test } from 'vitest'
+import userEvent from '@testing-library/user-event'
+import { screen, render, cleanup } from '@testing-library/react'
+import Header from '../Header'
+
+describe('Footer', () => {
+  afterEach(cleanup)
+
+  test('render component', () => {
+    const { container } = render(<Header />)
+    expect(container).toMatchSnapshot()
+  })
+
+  test('click Accedi button', async () => {
+    const { container } = render(<Header />)
+    const button = screen.getByRole('button', { name: 'Accedi' })
+    const user = userEvent.setup()
+    await user.click(button)
+  })
```

Comment:
Questo test non ha reali controlli (ex: `expect`..), deve essere in questo modo o è stata una dimenticanza?

Reply:
The button doesn't have any expected behavior for now and we will continue the test when it is implemented